### PR TITLE
@anandaroop => [ArtistSeries] Fallback to a representative artwork if an image is not set

### PR DESCRIPTION
**Pull request by mzikherman**
Seems to work pretty well!

### CODE REVIEW

File: src/lib/stitching/gravity/stitching.ts [hunk 1]
```diff
@@ -142,10 +143,31 @@ export const gravityStitchingEnvironment = (
...
+              context.imageData = {
+                image_url,
+                original_width,
+                original_height,
+              }
+            } else if (representativeArtworkID) {
+              const { artworkLoader } = context
+              const { images } = await artworkLoader(representativeArtworkID)
+              context.imageData = normalizeImageData(getDefault(images))
+            }
```
Comment (anandaroop) [inline src/lib/stitching/gravity/stitching.ts:170]: 👌 Niiice!

**Comment by mzikherman** [comment]
Yes definitely! Meant to add a spec to the previous PR with the initial `image` resolving.

**Comment by mzikherman** [comment]
Updated with specs!

**Comment by mzikherman** [comment]
gonna #mergeongreen to unblock the front-end stuff!

**Pull request merged**